packages/vuepress/vuepress-theme-titanium/components/PageEdit.vue

```diff
@@ -138,14 +138,14 @@ export default {
   .edit-link
     display inline-block
     a
-      color lighten($textColor, 25%)
+      color #006580
```

Comment:
I assume this came from the `primary` color in the Axway style guide? Can we assign it (and others) to known variable names and reference those?

https://zeroheight.com/3bf9d496d/p/23c5a7-design-tokens